apigw-lambda-bedrock-cdk-java/infrastructure/src/main/java/com/example/InfrastructureStack.java

```diff
@@ -0,0 +1,54 @@
+package com.example;
+
+import software.amazon.awscdk.App;
+import software.amazon.awscdk.Duration;
+import software.amazon.awscdk.Stack;
+import software.amazon.awscdk.StackProps;
+import software.amazon.awscdk.services.apigateway.LambdaIntegration;
+import software.amazon.awscdk.services.apigateway.RestApi;
+import software.amazon.awscdk.services.iam.PolicyStatement;
+import software.amazon.awscdk.services.iam.PolicyStatementProps;
+import software.amazon.awscdk.services.lambda.Code;
+import software.amazon.awscdk.services.lambda.Function;
+import software.amazon.awscdk.services.lambda.Runtime;
+import software.constructs.Construct;
+
+import java.util.List;
+
+public class InfrastructureStack extends Stack {
+    public InfrastructureStack(final App parent, final String id) {
+        this(parent, id, null);
+    }
+
+    public InfrastructureStack(final Construct parent, final String id, final StackProps props) {
+        super(parent, id, props);
+
+        // Create policy to allow Lambda call Bedrock
+        PolicyStatement invokeModelPolicy = new PolicyStatement(PolicyStatementProps.builder()
+                .actions(List.of("bedrock:InvokeModel"))
+                .resources(List.of("arn:aws:bedrock:" + getRegion() + "::foundation-model/anthropic.claude-v2"))
+                .build());
+
+
+        // Create a Lambda function
+        Function lambdaFunction = Function.Builder.create(this, "LambdaBedrockFunction")
```

Comment:
Suggest explicitly adding the CW Logs group as a resource so it can be cleaned up when deleting the stack.